gslib/commands/cp.py

```diff
@@ -631,147 +615,141 @@
-                 will perform an additional GET request to check if an item
-                 exists before attempting to upload the data. This will save
-                 retransmitting data, but the additional HTTP requests may make
+                 destination are not overwritten. Any items that are skipped
+                 by this option are reported as skipped. gsutil
+                 performs an additional GET request to check if an item
+                 exists before attempting to upload the data. This saves gsutil
+                 from retransmitting data, but the additional HTTP requests may make
                  small object transfers slower and more expensive.
 
-  -p             Causes ACLs to be preserved when copying in the cloud. Note
-                 that this option has performance and cost implications when
-                 using  the XML API, as it requires separate HTTP calls for
-                 interacting with ACLs. (There are no such performance or cost
-                 implications when using the -p option with the JSON API.) The
-                 performance issue can be mitigated to some degree by using
-                 gsutil -m cp to cause parallel copying. Note that this option
-                 only works if you have OWNER access to all of the objects that
-                 are copied.
-
-                 You can avoid the additional performance and cost of using
-                 cp -p if you want all objects in the destination bucket to end
-                 up with the same ACL by setting a default object ACL on that
-                 bucket instead of using cp -p. See "gsutil help defacl".
-
-                 Note that it's not valid to specify both the -a and -p options
+  -p             Enables ACLs to be preserved when copying in the cloud. Note
+                 that this option has performance and cost implications only when
+                 using the XML API, as it requires separate HTTP calls for
+                 interacting with ACLs. You can mitigate this 
+                 performance issue using ``gsutil -m cp`` to perform parallel
+                 copying. Note that this option only works if you have OWNER access
+                 to all objects that are copied. If you want all objects in the
+                 destination bucket to end up with the same ACL, you can avoid these
+                 performance issues by setting a default object ACL on that bucket
+                 instead of using ``cp -p``. See "gsutil help defacl".
+
+                 Note that it's not valid to specify both the ``-a`` and ``-p`` options
                  together.
 
-  -P             Causes POSIX attributes to be preserved when objects are
-                 copied. With this feature enabled, gsutil cp will copy fields
-                 provided by stat. These are the user ID of the owner, the group
-                 ID of the owning group, the mode (permissions) of the file, and
```

Comment:
Recommend retaining "mode" here, or removing reference to it in the next paragraph

Reply:
Done.